markets/mode/commons.ts

```diff
@@ -115,9 +115,10 @@ export const CommonsConfig: ICommonConfiguration = {
   ChainlinkAggregator: {
     [eModeNetwork.mode]: {
       USDT: '0xa204F4A3d167dF2621EF092915dC4D0777f8A83C',
+      USDC: '0xa204F4A3d167dF2621EF092915dC4D0777f8A83C',
```

Comment:
Why the addresses of USDT and USDC are the same?

Reply:
it's just for testing

Reply:
I see